people.rkt

```diff
@@ -687,6 +725,10 @@
                         #:dissertation "http://ccs.neu.edu/home/types/resources/pdf/g-dissertation-2020.pdf"
                         #:extra "(Brown University)"
                         #:personal-site "http://ccs.neu.edu/home/types/")
+              @(alumnus "Daniel Patterson" #:year 2022
+                        #:dissertation "https://dbp.io/pubs/2022/dbp-dissertation.pdf"
```

Comment:
Time for a blog post? (Leif too)

Reply:
You could cross post: https://blog.visr.pl/posts/use-deps/

Reply:
Oh...wait, I see what you were going for. nvm. :) But ya, I should probably be there...unless you want me to stay listed as a PhD student forever. :P )

Reply:
Yes, sorry! I wasn't sure when alum status officially kicks in—I was just going based off your website. Do you have a preferred link do your dissertation, or should I use the [ProQuest](https://www.proquest.com/docview/2719410129/abstract/8E84388F01D348FDPQ/1?accountid=12826) one? I'm not sure if that link works for everyone.

Reply:
Not @LeifAndersen, but apparently ProQuest watermarks if you don't have an institutional subscription :( (I found that out when sending stuff for the JFP dissertation thingy). Personal webpages have their issues, but at least they have the real damn thing...

Reply:
> Time for a blog post? (Leif too)

Once a blog editor, always the blog editor